services/web/server/tests/integration/test_exporter.py

```diff
@@ -262,11 +270,22 @@ def replace_uuids_with_sequences(project: Dict[str, Any]) -> Dict[str, Any]:
     return project
 
 
-def dict_without_keys(dict_data: Dict[str, Any], keys: Set[str]) -> Dict[str, Any]:
-    result = deepcopy(dict_data)
-    for key in keys:
-        result.pop(key, None)
-    return result
+def dict_without_keys(
+    dict_data: Dict[str, Any], skipped_keys: Set[str]
+) -> Dict[str, Any]:
+    def _delete_keys_from_dict(
+        dictionary: Dict[str, Any], keys: Set[str]
+    ) -> Dict[str, Any]:
+        modified_dict = {}
+        for key, value in dictionary.items():
+            if key not in keys:
+                if isinstance(value, dict):
+                    modified_dict[key] = _delete_keys_from_dict(value, keys)
+                else:
+                    modified_dict[key] = deepcopy(value)
+        return modified_dict
+
+    return _delete_keys_from_dict(dict_data, skipped_keys)
```

Comment:
I guess this is multi level key ignore.
Could you remove the inner function definition as it seems to have no benefit?

Reply:
not sure I understand that one?

Reply:
`_delete_keys_from_dict` is defined inside `dict_without_keys` and just called. it can be removed.

Reply:
ah right. will do thx!